### PR TITLE
Reporting style

### PR DESCRIPTION
**Pull request by gislenius**
fixes https://github.com/28msec/cellstore/issues/917

### CODE REVIEW

**Comment by gislenius** [comment]
popup

File: data/reports/secxbrl/COMID-BSC-CF1-ISM-IEMIX-OILY.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
 [ {
   "_id" : "COMID-BSC-CF1-ISM-IEMIX-OILY-GeneralInformation", 
   "Owner" : "admin@28.io", 
+  "Description" : "Report frame against the style COMID-BSC-CF1-ISM-IEMIX-OILY",
```
Comment (dknochen) [inline data/reports/secxbrl/COMID-BSC-CF1-ISM-IEMIX-OILY.json]: could you also easily add a url that points to charles definition of the report frames? If not this is already an improvement. I think, everybody will immediately come up with the question what a report frame actually is. Its not self explanatory.
what do you think?

**Comment by dknochen** [comment]
thank you for doing this improvement :+1: :)

**Comment by gislenius** [comment]
Done!

**Comment by dknochen** [comment]
@gislenius the tests seem to fail. Could you take a look?

**Comment by gislenius** [comment]
I uploaded a corrupt file.

**Comment by dknochen** [comment]
perfect! thank you

**Pull request merged**